src/base/search/searchpluginmanager.cpp

```diff
@@ -54,10 +55,30 @@
 
 namespace
 {
-    inline void removePythonScriptIfExists(const QString &scriptPath)
+    void clearPythonCache(const QString &path)
     {
-        Utils::Fs::forceRemove(scriptPath);
-        Utils::Fs::forceRemove(scriptPath + "c");
+        // remove python cache artifacts in `path` and subdirs
+
+        QStringList dirs = {path};
+
+        QDirIterator iter {path, (QDir::AllDirs | QDir::NoDotAndDotDot), QDirIterator::Subdirectories};
+        while (iter.hasNext())
+            dirs += iter.next();
+
+        // python 3: remove "__pycache__" folders
+        for (const QString &dir : dirs) {
+            if (dir.endsWith("__pycache__"))
+                Utils::Fs::removeDirRecursive(dir);
+        }
+
+        // python 2: remove "*.pyc" files
```

Comment:
@Chocobo1:  FYI, Python3 can use `.pyc` files as well.

Reply:
I thought qbt usage falls in the default case?
Anyway, any suggestions here?

Reply:
@Chocobo1:  I just think that you might want to note that some methods of manually triggering compilation in Python3 can also result in the compiled files being outside `pycache`…  

Since this is more of a nitpick than anything else, feel free to do as you wish.

Reply:
OK and thanks I wasn't really aware of that, I think the current code can catch all pyc files in the directory and sub-dirs.